src/App/helpers/jwt.js

```diff
@@ -0,0 +1,25 @@
+import 'dotenv/config';
```

Comment:
Please update the file to export a class instead of individual functions and make sure you are using it not just having it.